src/components/cli-tutor-mode/CliTutorMode.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'redux-bundler-react'
+
+// Components
+import { Modal, ModalBody, ModalActions } from '../modal/Modal'
+import CopyIcon from '../../icons/CopyIcon'
+import Button from '../button/Button'
+import Overlay from '../overlay/Overlay'
+import Shell from '../shell/Shell'
+import StrokeDownload from '../../icons/StrokeDownload'
+
+const CliTutorialModal = ({ command, t, onLeave, className, downloadConfig, ...props }) => {
+  const onClickCopyToClipboard = async (command) => {
+    await navigator.clipboard.writeText(command)
+  }
+
+  return (
+    <Modal {...props} className={className} onCancel={onLeave} style={{ maxWidth: '40em' }}>
+      <ModalBody icon={CopyIcon}>
+        <p className='charcoal w-80 center'>{t('copy the command to use it with ipfs command line interface')}</p>
+        <div>
+          <Shell title="terminal shell">
```

Comment:
```suggestion
          <Shell title="Shell">
```
Just "Shell" is fine here. Use the capital "S" in case we decide not to use CSS to uppercase these lines in future.